addons/website_event_track/views/event_track_templates_list.xml

```diff
@@ -52,7 +52,7 @@
                     </a>
                 </div>
             </div>
-            <div class="o_wevent_search d-flex w-100 w-lg-auto">
+            <div class="o_wevent_search o_wevent_event_search_box_track d-flex w-100 w-lg-auto">
```

Comment:
Could you rename `o_wevent_event_track_search_box` ? As it is better to keep the `_search_box` part at the end :slightly_smiling_face: